shared/build.gradle.kts

```diff
@@ -81,6 +86,7 @@ kotlin {
                 implementation(Dependency.KOTEST_FRAMEWORK)
                 implementation(Dependency.KOTEST_ASSERTIONS)
                 implementation(Dependency.KOTEST_PROPERTY)
+                implementation("app.cash.turbine:turbine:0.12.1")
```

Comment:
Do we have `Dependency` definition for this dependency?

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/103/commits/ba20a9aac1fce087a58244159f5a71b096cd60ed

I didn't do a self-review for this PR. Will improve later.